qiita_ware/test/test_ebi.py

```diff
@@ -557,116 +605,251 @@ def test_generate_demultiplexed_fastq(self):
 </STUDY_SET>
 """ % {'organization_prefix': qiita_config.ebi_organization_prefix}
 
-STUDYXML_PMIDS = """<?xml version="1.0" encoding="UTF-8"?>
-<STUDY_SET xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noName\
-spaceSchemaLocation="ftp://ftp.sra.ebi.ac.uk/meta/xsd/sra_1_3/SRA.study.xsd">
-  <STUDY alias="%(organization_prefix)s_ppdid_2" center_name="CCME-\
-COLORADO">
-    <DESCRIPTOR>
-      <STUDY_TITLE>
-        Identification of the Microbiomes for Cannabis Soils
-      </STUDY_TITLE>
-      <STUDY_TYPE existing_study_type="Metagenomics"/>
-      <STUDY_ABSTRACT>
-        This is a preliminary study to examine the microbiota associated with \
-the Cannabis plant. Soils samples from the bulk soil, soil associated with \
-the roots, and the rhizosphere were extracted and the DNA sequenced. Roots \
-from three independent plants of different strains were examined. These roots \
-were obtained November 11, 2011 from plants that had been harvested in the \
-summer. Future studies will attempt to analyze the soils and rhizospheres \
-from the same location at different time points in the plant lifecycle.
-      </STUDY_ABSTRACT>
-    </DESCRIPTOR>
-    <STUDY_LINKS>
-      <STUDY_LINK>
-        <XREF_LINK>
-          <DB>PUBMED</DB>
-          <ID>123456</ID>
-        </XREF_LINK>
-      </STUDY_LINK>
-      <STUDY_LINK>
-        <XREF_LINK>
-          <DB>PUBMED</DB>
-          <ID>7891011</ID>
-        </XREF_LINK>
-      </STUDY_LINK>
-    </STUDY_LINKS>
-  </STUDY>
-</STUDY_SET>
-""" % {'organization_prefix': qiita_config.ebi_organization_prefix}
-
-EXPERIMENTXML = """<?xml version="1.0" encoding="UTF-8"?>
+EXPERIMENTXML = """
 <EXPERIMENT_SET xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:no\
 NamespaceSchemaLocation="ftp://ftp.sra.ebi.ac.uk/meta/xsd/sra_1_3/SRA.\
 experiment.xsd">
-  <EXPERIMENT alias="%(organization_prefix)s_ppdid_2:test1" center_name=\
-"CCME-COLORADO">
-    <TITLE>%(organization_prefix)s_ppdid_2:test1</TITLE>
-    <STUDY_REF refname="%(organization_prefix)s_ppdid_2"/>
+  <EXPERIMENT alias="%(organization_prefix)s_ppdid_2:1.SKB2.640194" \
+center_name="CCME-COLORADO">
```

Comment:
Is the center_name now in the qiita_config? If so, then the expected text should be changed to use the center_name that is specified there rather than assuming CCME-COLORADO